### DISCUSSION TITLE
Will Kurtosis be able to run my package remotely from a private Github repository?

### DISCUSSION THREAD

**Discussion by guisellaDes** [kurtosis-tech/kurtosis#1275] [Q&A] [answered]
Will Kurtosis be able to run my package remotely from a private Github repository?

**Comment by leeederek** [comment, marked as answer]
Hey @guisellaDes: No, Kurtosis is currently unable to run packages that reside in a private Github repository. Please file a [Github issue on our repository](https://github.com/kurtosis-tech/kurtosis/issues/new?assignees=&labels=feature+request&projects=&template=feature-request.yml) if you believe you need this workflow! Cheers!
